src/main/java/codeu/model/data/Event.java

```diff
@@ -0,0 +1,34 @@
+package codeu.model.data;
+
+import java.time.Instant;
+import java.util.UUID;
+import java.util.List;
+import java.util.ArrayList;
+
+public class Event implements Comparable<Event> {
+  private final UUID id;
+  private final Instant creation;
+
+  public Event(UUID id, Instant creation) {
+    this.id = id;
+    this.creation = creation;
+  }
+
+  /** Returns the ID of this Event. */
+  public UUID getId() {
+    return id;
+  }
+
+  /** Returns the creation time of this Event. */
+  public Instant getCreationTime(){
+    return creation;
+  }
+
+  @Override
+  public int compareTo(Event otherEvent) {
+    if (this.getCreationTime().isBefore(otherEvent.getCreationTime())) {
+      return -1;
+    } else
```

Comment:
Nit: you should return 0 if they are equal.

Nit: most coding styles I've seen use braces everywhere. Note that we don't have an official coding style, though.